mirgecom/fluid.py

```diff
@@ -427,9 +426,9 @@ def species_mass_fraction_gradient(discr, cv, grad_cv):
     """
     nspecies = len(cv.species_mass)
     y = cv.species_mass / cv.mass
-    return (1/cv.mass)*make_obj_array([grad_cv.species_mass[i]
-                                       - y[i]*grad_cv.mass
-                                       for i in range(nspecies)])
+    return (1/cv.mass)*np.array([grad_cv.species_mass[i]
+                                 - y[i]*grad_cv.mass
+                                 for i in range(nspecies)], dtype=object)
```

Comment:
And this one can be written as
```suggestion
    return (grad_cv.species_mass - np.outer(y, grad_cv.mass))/cv.mass
```
?